statsmodels/duration/tests/test_survfunc.py

```diff
@@ -238,3 +238,50 @@ def test_plot_km():
 
     if pdf_output:
         pdf.close()
+
+
+def test_weights1():
+    """
```

Comment:
No docstring for test function, use `#` for code comment instead of triple quotes.
(nosetests uses docstring if available as "title")